src/harness/testcases/testdata/esc_sensor_record_1.json

```diff
@@ -0,0 +1,1465 @@
+{
```

Comment:
We   have removed this file now and used the existing esc_sensor_record_0.json   instead of creating a new one. The new file actually contained some   additional parameters that were needed for interference calculation. I   understand that they are not used in this PR and they can get added when the   related code gets submitted. At present this updated format is removed and   the existing record is used.